### PR TITLE
✨NEW: Landing page for the tutorial

### PR DESCRIPTION
**Pull request by mbercx**
Largely adapted from the previous version. The link to my intro presentation
still needs to be added, since I haven't made it yet.

Also cleaned up the `test.md` file, which was mainly used to test the new MyST
syntax.

### CODE REVIEW

**Comment by mbercx** [comment]
@ramirezfranciscof note that the PR is still blocked by #380 (since it builds on top of it).

**Comment by mbercx** [comment]
All rebased and fixed up, Capt'n @ramirezfranciscof!

File: docs/index.rst [hunk 1]
```diff
@@ -1,61 +1,138 @@
...
-   sections/getting_started/index
-   sections/running_processes/index
-   sections/managing_data/index
-   sections/writing_workflows/index
-   sections/creating_plugins/index
+Contributed talks
+-----------------
+
+Supplementing the presentations above, there are also contributed talks from plugin developers and experienced AiiDA users.
+These can be watched at any time, but please try to watch them **before Friday July 9th**, as we have planned a Q&A with the plugin developers at the start of the first hands-on session of that day.
```
Comment (ramirezfranciscof) [inline docs/index.rst:96]: Maybe we can bold/highlight those that will be present at the Q&A?

**Review by ramirezfranciscof — APPROVED** [review]
All is good, just a small comment to consider

File: docs/index.rst (hunk 1 above)
Comment (mbercx) [reply docs/index.rst:96]: Sure, can you remind me again? Emanuele, Togo, Dominik, Leonid (but his talk isn't here yet), ... Anyone else?

Comment (mbercx) [reply docs/index.rst:96]: I'll update this later ;)

**Pull request merged**